Python/pywarpx/_libwarpx.py

```diff
@@ -478,7 +233,8 @@ def getistep(self, level=0):
             The refinement level to reference
         '''
 
-        return self.libwarpx_so.warpx_getistep(level)
+        warpx = self.libwarpx_so.get_instance()
+        return warpx.getistep(level)
```

Comment:
Code of this sort should be,
```suggestion
        return self.warpx.getistep(level)
```
right?